src/scales/linearScale.ts

```diff
@@ -53,7 +53,7 @@ export module Scales {
     }
 
     protected _niceDomain(domain: number[], count?: number): number[] {
-      return Utils.D3Scale.niceDomain(this._d3Scale, domain, count);
+      return this._d3Scale.copy().domain(domain).nice(count).domain();
```

Comment:
It depended on the D3 type `QuantitativeScale` existing, but that type no longer exists. Rather than have it take some sort of union type, I thought it made more sense just to clone the implementation, in particular because the signature of the `nice()` method is different on different scales.